doc/man1/genpkey.pod

```diff
@@ -63,13 +63,27 @@ options.
 
 Public key algorithm to use such as RSA, DSA or DH. If used this option must
 precede any B<-pkeyopt> options. The options B<-paramfile> and B<-algorithm>
-are mutually exclusive.
+are mutually exclusive. Engines may add algorithms in addition to the standard
+built-in ones.
+
+Valid built-in algorithm names for private key generation are RSA, RSA-PSS, EC,
+X25519, X448, ED25519 and ED448.
+
+Valid built-in algorithm names for parameter generation (see the B<-genparam>
+option) are DH, DSA and EC.
+
+Note that the algorithm name X9.42 DH may be used as a synonym for the DH
+algorithm. These are identical in every respect and do not indicate the type of
+parameters that will be generated. Use the B<dh_paramgen_type> option to indicate
+whether PKCS#3 or X9.42 DH parameters are required. See L<DH Parameter
```

Comment:
Does perldoc accept splitting across lines?  I think our doc-nits script doesn't, so maybe a newline before the L

Reply:
The "in every respect" rewrapped this text so the issue went away.